daprdocs/content/en/reference/cli/dapr-init.md

```diff
@@ -33,11 +33,14 @@ dapr init [flags]
 | `--namespace`, `-n`  |                      | `dapr-system` | The Kubernetes namespace to install Dapr in                                          |
 | `--runtime-version`  |                      | `latest`      | The version of the Dapr runtime to install, for example: `1.0.0`                     |
 | `--slim`, `-s`       |                      | `false`       | Exclude placement service, Redis and Zipkin containers from self-hosted installation |
+| `--image-registry`   |                      |               | Pulls container images required by dapr from given image registry                    |
 
 ### Examples
 
 #### Self-hosted environment
 
+Setup Dapr by pulling container images of Placement, Redis and Zipkin. By default these images will be pulled from Github container registry. To switch to docker as a default you can set `DAPR_DEFAULT_IMAGE_REGISTRY` as environment variable and value as `DOCKERHUB`
```

Comment:
```suggestion
Install Dapr by pulling container images for Placement, Redis and Zipkin. By default these images are pulled from Dapr Github container registry. To switch to DockerHub as the default registry, set the `DAPR_DEFAULT_IMAGE_REGISTRY` environment variable value to be `DOCKERHUB`
```

Reply:
Can you also update this reference document for environment variables topic https://docs.dapr.io/reference/environment/ saying this is used by the CLI.

Reply:
Now since we are defaulting back to dockerhub, we can value for this variable as `GHCR` to default to Github container registry. We can set only one value `GHCR` or `ghcr` and we can unset it to default back to dockerhub.